src/probnum/quad/solvers/_bayesian_quadrature.py

```diff
@@ -316,6 +310,10 @@ def integrate(
             If dimension of ``nodes`` or ``fun_evals`` is incorrect, or if their
             shapes do not match.
         """
+        # Get the rng
+        if isinstance(rng, get_args(IntLike)):
+            rng = np.random.default_rng(int(rng))
+
```

Comment:
```suggestion
```
See above, why I think this can be dangerous and should be avoided.

Reply:
removed since obsolete.